synfig-studio/src/gui/dialogs/dialog_setup.cpp

```diff
@@ -501,6 +501,13 @@ Dialog_Setup::create_render_page(PageInfo pi)
 	pi.grid->attach(image_sequence_separator, 1, row, 1, 1);
 	image_sequence_separator.set_hexpand(true);
 
+	// render - cores
+	attach_label(pi.grid, _("Limit render to number of cores"), ++row);
+	pi.grid->attach(core_render, 1, row, 1, 1);
```

Comment:
`core_render` is really a bad name choice :P It doesn't mean what it stores/is.

Reply:
I know they are bad names [ Sorry I forgot to write it :disappointed:   ] but I just wanted to verify logic. because I am new here.
Sorry for the trouble in pointing out the bad name.

Reply:
No problem here. Logic is looks correct. Also you need to check CI builds. Currently they fails with error.

Reply:
@SushantAA I'm sorry if it sounded harsh, it wasn't my intention. It was my last comment about variable name and I explained it better in previous comments, so I thought I could be more concise. I was wrong :(

Reply:
@rodolforg nothing was harsh, it's fine  :smiley: